dbms/src/Interpreters/Join.cpp

```diff
@@ -456,6 +457,9 @@ void Join::insertFromBlock(const Block & block, size_t stream_index)
                     {
                         join_partition->addMemoryUsage(byte_after_insert - byte_before_insert);
                     }
+                    auto ret = hash_join_spill_context->updatePartitionRevocableMemory(force_spill_partition_blocks, i, join_partition->revocableBytes());
```

Comment:
Why is it necessary to call `updatePartitionRevocableMemory` here, and what is the difference with L447?

Reply:
Because the size of hash table is increased.

Reply:
I will refine the code to handle spill/unspill partition independently.